nemo_text_processing/text_normalization/en/data/address/address_words.tsv

```diff
@@ -1,13 +1,14 @@
-St	Street
-Expy	Expressway
-Fwy	Freeway
-Hwy	Highway
-Dr	Drive
-Ct	Court
-Ave	Avenue
-Av	Avenue
-CIR	Circle
-Blvd	Boulevard
-Alley
-Way
-Jct	Junction
\ No newline at end of file
+st	Street
+street	Street
+expy	Expressway
```

Comment:
why removing casing?

Reply:
because we're getting all possible casing [here](https://github.com/NVIDIA/NeMo/blob/9cd221951d14f984eb3673ada426ea8096419951/nemo_text_processing/text_normalization/en/taggers/measure.py#L297)